include/SH3/graphics/texture.hpp

```diff
@@ -1,56 +1,89 @@
-/*++
+/** @file
+ *
+ *  Functions and structures to load a texture from an arc section
+ *
+ *  \copyright 2016  Palm Studios
```

Comment:
We should choose either consistently `@tag` or `\tag`. I made the same error in my doxygen branch...